spec/dry/core/descendants_tracker_spec.rb

```diff
@@ -18,6 +18,6 @@ class Grandchild < Child
   end
 
   it "tracks descendants" do
-    expect(Test::Parent.descendants).to eql([Test::Grandchild, Test::Child])
+    expect(Test::Parent.descendants).to eql([Test::Child, Test::Grandchild])
```

Comment:
Looks like a breaking change?

Reply:
yeah, it'll be. But I think it's fine since it's now consistent with `subclasses` and [other implementations](https://github.com/rails/rails/pull/43951/files#diff-07b79ac29ae5d66631a64303088914c230cc3fa9c66625e18feed7ee6dab0901R9). Application-wise this shouldn't be a problem for all/almost all cases IMO. Nobody declared the order of the collection, it could be a set as well.

Reply:
we're still 0.x btw

Reply:
yeah I know, just pointing it out, as it's something worth mentioning in the changelog